std/algorithm/setops.d

```diff
@@ -404,7 +404,7 @@ if (ranges.length >= 2 &&
                     r = ranges[i].save; // rollover
             }
         }
-        @property Result save() scope return
+        @property Result save() return scope
```

Comment:
Should it be the same?

Reply:
It's not the same, see https://dlang.org/changelog/2.099.0.html#22541